src/pages/ExamsPage/components/ExternalReviewDashboard.test.jsx

```diff
@@ -21,6 +21,12 @@ describe('ExternalReviewDashboard', () => {
   });
   it('includes an iframe with the correct url for the current exam', () => {
     render(<ExternalReviewDashboard exam={testExam} />);
-    expect(screen.getByTitle('lti_tool')).toHaveAttribute('src', 'http://test.org/lti/exam/3/instructor_tool');
+    expect(screen.getByText('View resource').toBeInTheDocument());
+    expect(screen.getByTitle('lti_link')).toHaveAttribute('href', 'http://test.org/lti/exam/3/instructor_tool');
   });
+  // todo: add back after embed vs new tab is a configurable option.
```

Comment:
Commenting this out for now because we always want to open in a new tab. Let me know if I should keep/approach differently.